test/index.js

```diff
@@ -564,11 +564,16 @@ lab.experiment('MongoModels Proxied Methods', () => {
                 $set: { isCool: true }
             };
 
-            SubModel.updateOne(filter, update, (err, count) => {
+            SubModel.updateOne(filter, update, (err, count, result) => {
 
                 Code.expect(err).to.not.exist();
                 Code.expect(count).to.be.a.number();
                 Code.expect(count).to.equal(1);
+                Code.expect(result).to.be.an.object();
```

Comment:
Let's only confirm `result` is an object and not that it includes specific properties (again incase they change the document in the future).